pkg/cmd/server/origin/audit.go

```diff
@@ -0,0 +1,24 @@
+package origin
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/golang/glog"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+func basicAuditHandler(handler http.Handler, contextMapper kapi.RequestContextMapper) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		ctx, _ := contextMapper.Get(req)
+		user, _ := kapi.UserFrom(ctx)
+		namespace := kapi.NamespaceValue(ctx)
+		usernamespace := user.GetName()
+		if len(namespace) != 0 {
+			usernamespace = fmt.Sprintf("%s@%s", user.GetName(), namespace)
+		}
+		glog.Infof("%s - %s - %v", req.Method, usernamespace, req.URL)
```

Comment:
I'd like to suggest a standard prefix for all audit statements so they're easy to parse out of the log with a grep rather than finding all info messages.  Maybe `"AUDIT: <field>:%s,<field>:%s, ...">`.  Adding the field name in there will probably also save us a bit of explanation for those trying to parse it.  @soltysh WDYT?